Core/src/Propagator/SympyStepper.cpp

```diff
@@ -162,6 +162,8 @@ Result<double> SympyStepper::stepImpl(
             state.pars.template segment<1>(eFreeTime).data(),
             state.derivative.data(),
             state.covTransport ? state.jacTransport.data() : nullptr);
+    // Protect against division by zero
+    errorEstimate = std::max(1e-20, errorEstimate);
```

Comment:
Fine like that, but maybe `std::numeric_limits<double>::min()` would be less arbitrary...

Reply:
This is the magic number we use in the `EigenStepper` - good point to make it less magical. I will try to follow up on this in the future. I think the point of this is just to not make `1 / error` fail which should be the case for `min()` I guess?

Reply:
Hmm If I believe in [cppreference](https://en.cppreference.com/w/cpp/types/numeric_limits/min) this should work I think...

> For floating-point types with denormalization, min() returns the minimum positive normalized value.